code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -225,11 +225,26 @@
 	illustration = "heart"
 	foldable_result = null
 	mask_type = null
+	var/random_funny_internals = TRUE
+
+/obj/item/storage/box/survival/hug/PopulateContents()
+	..()
+	if(!random_funny_internals)
+		return
+	var/obj/item/internals = locate(internal_type) in src
+	qdel(internals)
+	var/which_funny_tank = pick(list(
+			/obj/item/tank/internals/emergency_oxygen/engi/clown/n2o,
+			/obj/item/tank/internals/emergency_oxygen/engi/clown/bz,
+			/obj/item/tank/internals/emergency_oxygen/engi/clown/helium,
+			))
+	new which_funny_tank(src)
```

Comment:
```suggestion
/obj/item/storage/box/survival/hug/PopulateContents()
	internal_type = pick(
			/obj/item/tank/internals/emergency_oxygen/engi/clown/n2o,
			/obj/item/tank/internals/emergency_oxygen/engi/clown/bz,
			/obj/item/tank/internals/emergency_oxygen/engi/clown/helium,
			)
	return ..()
```
If this doesn't work, then you're in for some "bug fixing" as Time-Green said.

Reply:
This worked and it a lot smoother than other things I tried.

Thank you Ghommie.